jobs/config.json

```diff
@@ -8443,7 +8443,7 @@
       "--kubeadm=ci",
       "--kubernetes-anywhere-kubernetes-version=ci/latest",
       "--provider=kubernetes-anywhere",
-      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --minStartupPods=8",
+      "--test_args=--ginkgo.focus=\\[Conformance\\]|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\] --ginkgo-parallel=30 --minStartupPods=8",
```

Comment:
I think it need to be its own flag, like https://github.com/kubernetes/test-infra/blob/master/jobs/config.json#L846

Reply:
ah, ok, will do